### ISSUE TITLE
Card fails with JSON error on work network

### ISSUE THREAD

**Issue by K-Money** [r-renato/ha-card-weather-conditions#31]
When viewing the lovelace frontend from within my work's network, the Climacell Weather card fails to load and gives this error:

`Unexpected token < in JSON at position 0`

I can view the card fine from the local network as well as outside of the local network. It's only when within the work network that this specific card breaks. I suspect that due to my work's IT policies, some kind of call/request made by the card is being blocked that isn't otherwise blocked out in the wild. Although I don't understand A) Why the card would make a call when all the data is already available in the sensors (the sensors all work fine) B) Why the card fails in this big red way rather than just not appearing or not showing the small error'd section (e.g. not showing a temp value).

Note: Screenshot was from a couple versions ago so the entity names might be slightly out of date but the problem persists through the current version.

On work network:
![ClimacellError](https://user-images.githubusercontent.com/5406336/87236508-7813c780-c39f-11ea-8834-b4c85fb640a5.JPG)

On local network or outside network:
![ClimacellNormal](https://user-images.githubusercontent.com/5406336/87236547-d476e700-c39f-11ea-9c7d-6d1e9d4f79ec.JPG)

**Comment by r-renato** [comment]
Hi, 
the first sensor is sensor.<name>_0d, see the example:

```
type: custom:ha-card-weather-conditions
  name: "cc_test"
  language: it
  animation: true
  weather:
    icons_model: climacell
    forecast:
      meteogram: camera.cc_test_c2
      icons:
        day_1: sensor.cc_test_weather_condition_0d
        day_2: sensor.cc_test_weather_condition_1d
        day_3: sensor.cc_test_weather_condition_2d
        day_4: sensor.cc_test_weather_condition_3d
        day_5: sensor.cc_test_weather_condition_4d
      temperature_high:
        day_1: sensor.cc_test_temperature_max_0d
        day_2: sensor.cc_test_temperature_max_1d
        day_3: sensor.cc_test_temperature_max_2d
        day_4: sensor.cc_test_temperature_max_3d
        day_5: sensor.cc_test_temperature_max_4d
      temperature_low:
        day_1: sensor.cc_test_temperature_min_0d
        day_2: sensor.cc_test_temperature_min_1d
        day_3: sensor.cc_test_temperature_min_2d
        day_4: sensor.cc_test_temperature_min_3d
        day_5: sensor.cc_test_temperature_min_4d
      precipitation_probability:
        day_1: sensor.cc_test_precipitation_probability_0d
        day_2: sensor.cc_test_precipitation_probability_1d
        day_3: sensor.cc_test_precipitation_probability_2d
        day_4: sensor.cc_test_precipitation_probability_3d
        day_5: sensor.cc_test_precipitation_probability_4d
      precipitation_intensity:
        day_1: sensor.cc_test_precipitation_max_0d
        day_2: sensor.cc_test_precipitation_max_1d
        day_3: sensor.cc_test_precipitation_max_2d
        day_4: sensor.cc_test_precipitation_max_3d
        day_5: sensor.cc_test_precipitation_max_4d
```

Thanks, bye
Renato

**Comment by K-Money** [comment]
> Hi,
> the first sensor is sensor._0d, see the example:
> 
> ```
> type: custom:ha-card-weather-conditions
>   name: "cc_test"
>   language: it
>   animation: true
>   weather:
>     icons_model: climacell
>     forecast:
>       meteogram: camera.cc_test_c2
>       icons:
>         day_1: sensor.cc_test_weather_condition_0d
>         day_2: sensor.cc_test_weather_condition_1d
>         day_3: sensor.cc_test_weather_condition_2d
>         day_4: sensor.cc_test_weather_condition_3d
>         day_5: sensor.cc_test_weather_condition_4d
>       temperature_high:
>         day_1: sensor.cc_test_temperature_max_0d
>         day_2: sensor.cc_test_temperature_max_1d
>         day_3: sensor.cc_test_temperature_max_2d
>         day_4: sensor.cc_test_temperature_max_3d
>         day_5: sensor.cc_test_temperature_max_4d
>       temperature_low:
>         day_1: sensor.cc_test_temperature_min_0d
>         day_2: sensor.cc_test_temperature_min_1d
>         day_3: sensor.cc_test_temperature_min_2d
>         day_4: sensor.cc_test_temperature_min_3d
>         day_5: sensor.cc_test_temperature_min_4d
>       precipitation_probability:
>         day_1: sensor.cc_test_precipitation_probability_0d
>         day_2: sensor.cc_test_precipitation_probability_1d
>         day_3: sensor.cc_test_precipitation_probability_2d
>         day_4: sensor.cc_test_precipitation_probability_3d
>         day_5: sensor.cc_test_precipitation_probability_4d
>       precipitation_intensity:
>         day_1: sensor.cc_test_precipitation_max_0d
>         day_2: sensor.cc_test_precipitation_max_1d
>         day_3: sensor.cc_test_precipitation_max_2d
>         day_4: sensor.cc_test_precipitation_max_3d
>         day_5: sensor.cc_test_precipitation_max_4d
> ```
> 
> Thanks, bye
> Renato

Aye, the screenshot was from a couple versions ago but I keep the yaml up to date from the docs. The problem exists with correct configuration. Here's an updated screenshot.
![ClimacellError2](https://user-images.githubusercontent.com/5406336/87828272-6c0d8700-c831-11ea-994f-c0277098c697.JPG)

**Comment by r-renato** [comment]
Hi, can you put here your card config?
Thanks,
Renato

**Comment by K-Money** [comment]
Here's the current card config:

```
alert:
  fire_risk:
    entity: sensor.cc_climacell_fire_index
    icon: 'mdi:campfire'
    max: 100
    min: 1
    show_if_ge: 50
animation: true
language: en
type: 'custom:ha-card-weather-conditions'
weather:
  current:
    current_conditions: sensor.cc_climacell_weather_condition
    forecast: true
    humidity: sensor.netatmo_unit_01_backyard_humidity
    sun: sun.sun
    temperature: sensor.netatmo_unit_01_backyard_temperature
    wind_bearing: sensor.cc_climacell_wind_direction
    wind_speed: sensor.cc_climacell_wind_speed
  forecast:
    icons:
      day_1: sensor.cc_climacell_weather_condition_0d
      day_2: sensor.cc_climacell_weather_condition_1d
      day_3: sensor.cc_climacell_weather_condition_2d
      day_4: sensor.cc_climacell_weather_condition_3d
      day_5: sensor.cc_climacell_weather_condition_4d
    precipitation_intensity:
      day_1: sensor.cc_climacell_precipitation_max_0d
      day_2: sensor.cc_climacell_precipitation_max_1d
      day_3: sensor.cc_climacell_precipitation_max_2d
      day_4: sensor.cc_climacell_precipitation_max_3d
      day_5: sensor.cc_climacell_precipitation_max_4d
    precipitation_probability:
      day_1: sensor.cc_climacell_precipitation_probability_0d
      day_2: sensor.cc_climacell_precipitation_probability_1d
      day_3: sensor.cc_climacell_precipitation_probability_2d
      day_4: sensor.cc_climacell_precipitation_probability_3d
      day_5: sensor.cc_climacell_precipitation_probability_4d
    temperature_high:
      day_1: sensor.cc_climacell_temperature_max_0d
      day_2: sensor.cc_climacell_temperature_max_1d
      day_3: sensor.cc_climacell_temperature_max_2d
      day_4: sensor.cc_climacell_temperature_max_3d
      day_5: sensor.cc_climacell_temperature_max_4d
    temperature_low:
      day_1: sensor.cc_climacell_temperature_min_0d
      day_2: sensor.cc_climacell_temperature_min_1d
      day_3: sensor.cc_climacell_temperature_min_2d
      day_4: sensor.cc_climacell_temperature_min_3d
      day_5: sensor.cc_climacell_temperature_min_4d
  icons_model: climacell
```

**Comment by r-renato** [comment]
Please, try this configuration.
Thanks,
Renato

```
              - type: 'custom:ha-card-weather-conditions'
                animation: true
                language: en
                alert:
                  fire_risk:
                    entity: sensor.cc_climacell_fire_index
                    icon: 'mdi:campfire'
                    max: 100
                    min: 1
                    show_if_ge: 50
                weather:
                  current:
                    current_conditions: sensor.cc_climacell_weather_condition
                    forecast: true
                    humidity: sensor.netatmo_unit_01_backyard_humidity
                    sun: sun.sun
                    temperature: sensor.netatmo_unit_01_backyard_temperature
                    wind_bearing: sensor.cc_climacell_wind_direction
                    wind_speed: sensor.cc_climacell_wind_speed
                  forecast:
                    icons:
                      day_1: sensor.cc_climacell_weather_condition_0d
                      day_2: sensor.cc_climacell_weather_condition_1d
                      day_3: sensor.cc_climacell_weather_condition_2d
                      day_4: sensor.cc_climacell_weather_condition_3d
                      day_5: sensor.cc_climacell_weather_condition_4d
                    precipitation_intensity:
                      day_1: sensor.cc_climacell_precipitation_max_0d
                      day_2: sensor.cc_climacell_precipitation_max_1d
                      day_3: sensor.cc_climacell_precipitation_max_2d
                      day_4: sensor.cc_climacell_precipitation_max_3d
                      day_5: sensor.cc_climacell_precipitation_max_4d
                    precipitation_probability:
                      day_1: sensor.cc_climacell_precipitation_probability_0d
                      day_2: sensor.cc_climacell_precipitation_probability_1d
                      day_3: sensor.cc_climacell_precipitation_probability_2d
                      day_4: sensor.cc_climacell_precipitation_probability_3d
                      day_5: sensor.cc_climacell_precipitation_probability_4d
                    temperature_high:
                      day_1: sensor.cc_climacell_temperature_max_0d
                      day_2: sensor.cc_climacell_temperature_max_1d
                      day_3: sensor.cc_climacell_temperature_max_2d
                      day_4: sensor.cc_climacell_temperature_max_3d
                      day_5: sensor.cc_climacell_temperature_max_4d
                    temperature_low:
                      day_1: sensor.cc_climacell_temperature_min_0d
                      day_2: sensor.cc_climacell_temperature_min_1d
                      day_3: sensor.cc_climacell_temperature_min_2d
                      day_4: sensor.cc_climacell_temperature_min_3d
                      day_5: sensor.cc_climacell_temperature_min_4d
                  icons_model: climacell
```

**Comment by K-Money** [comment]
The config looks identical to me, just with the order of rows swapped around? I tried it anyways and update the climacell to the recent latest release as well, but none of it had any impact on the issue. It gives exactly the same behavior: totally fine on any network outside my work network, fails within my work network.

**Comment by r-renato** [comment]
Can you put here the browser javascript console?

**Comment by K-Money** [comment]
The Javascript browser console output shows this:

```
ha-card-weather-conditions.js:381  WEATHER-CONDITION-CARD  1.9.4
chunk.55cfcfe8bf5cea239c42.js:1 Uncaught (in promise) https://hassio.kmoneyserver.com/customcards/github/thomasloven/state-switch.js?track=true
cloudy.svg:1 Failed to load resource: the server responded with a status of 503 (Service Unavailable)
cloudy.svg:1 Failed to load resource: the server responded with a status of 404 (Not Found)
ha-card-weather-conditions.js:381  WEATHER-CONDITION-CARD  1.9.4 error setting right icons path.
ha-card-weather-conditions.js:381 Object
chunk.55cfcfe8bf5cea239c42.js:12717 card custom:ha-card-weather-conditions SyntaxError: Unexpected token < in JSON at position 0
    at JSON.parse (<anonymous>)
    at HTMLElement.setConfig (ha-card-weather-conditions.js:381)
    at s (chunk.55cfcfe8bf5cea239c42.js:12717)
    at chunk.55cfcfe8bf5cea239c42.js:12717
    at d (chunk.55cfcfe8bf5cea239c42.js:12717)
    at l (chunk.55cfcfe8bf5cea239c42.js:12717)
    at Kt (chunk.55cfcfe8bf5cea239c42.js:11940)
    at HTMLElement.value (chunk.55cfcfe8bf5cea239c42.js:23033)
    at HTMLElement.value (chunk.55cfcfe8bf5cea239c42.js:23460)
    at HTMLElement.<anonymous> (chunk.55cfcfe8bf5cea239c42.js:23033)
l @ chunk.55cfcfe8bf5cea239c42.js:12717
ha-card-weather-conditions.js:381 Object
chunk.55cfcfe8bf5cea239c42.js:12717 card custom:ha-card-weather-conditions SyntaxError: Unexpected token < in JSON at position 0
    at JSON.parse (<anonymous>)
    at HTMLElement.setConfig (ha-card-weather-conditions.js:381)
    at s (chunk.55cfcfe8bf5cea239c42.js:12717)
    at chunk.55cfcfe8bf5cea239c42.js:12717
    at d (chunk.55cfcfe8bf5cea239c42.js:12717)
    at l (chunk.55cfcfe8bf5cea239c42.js:12717)
    at Kt (chunk.55cfcfe8bf5cea239c42.js:11940)
    at HTMLElement.value (chunk.55cfcfe8bf5cea239c42.js:23033)
    at HTMLElement.value (chunk.55cfcfe8bf5cea239c42.js:23460)
    at HTMLElement.<anonymous> (chunk.55cfcfe8bf5cea239c42.js:23033)
l @ chunk.55cfcfe8bf5cea239c42.js:12717
```

In the console, this line:
`cloudy.svg:1 Failed to load resource: the server responded with a status of 503 (Service Unavailable)`
has this hyperlink:
https://hassio.kmoneyserver.com/local/community/ha-card-weather-conditions/icons/static/cloudy.svg
 and when I try to go to that link it's blocked by my company's firewall. However, the base URL isn't blocked as that's how I'm accessing the instance in the first place.

I suspect if the card itself is trying to make some kind of request independent of the component that it's getting blocked.

**Comment by r-renato** [comment]
Hi, I have increase the log level in the newly released version. Can you install the last version and put here the browser console logs?

Thanks, bye
Renato

**Comment by K-Money** [comment]
Hi, the log looks the same except for the one new devtools line at the end. Thanks!

```
ha-card-weather-conditions.js:381  WEATHER-CONDITION-CARD  1.9.5
chunk.55cfcfe8bf5cea239c42.js:1 Uncaught (in promise) https://hassio.kmoneyserver.com/customcards/github/thomasloven/state-switch.js?track=true
cloudy.svg:1 Failed to load resource: the server responded with a status of 503 (Service Unavailable)
cloudy.svg:1 Failed to load resource: the server responded with a status of 404 (Not Found)
ha-card-weather-conditions.js:381  WEATHER-CONDITION-CARD  1.9.5 error setting right icons path.
ha-card-weather-conditions.js:381 Object
chunk.55cfcfe8bf5cea239c42.js:12717 card custom:ha-card-weather-conditions SyntaxError: Unexpected token < in JSON at position 0
    at JSON.parse (<anonymous>)
    at HTMLElement.setConfig (ha-card-weather-conditions.js:381)
    at s (chunk.55cfcfe8bf5cea239c42.js:12717)
    at chunk.55cfcfe8bf5cea239c42.js:12717
    at d (chunk.55cfcfe8bf5cea239c42.js:12717)
    at l (chunk.55cfcfe8bf5cea239c42.js:12717)
    at Kt (chunk.55cfcfe8bf5cea239c42.js:11940)
    at HTMLElement.value (chunk.55cfcfe8bf5cea239c42.js:23033)
    at HTMLElement.value (chunk.55cfcfe8bf5cea239c42.js:23460)
    at HTMLElement.<anonymous> (chunk.55cfcfe8bf5cea239c42.js:23033)
l @ chunk.55cfcfe8bf5cea239c42.js:12717
ha-card-weather-conditions.js:381 Objectcard_config: {alert: {…}, animation: true, language: "en", type: "custom:ha-card-weather-conditions", weather: {…}}__proto__: Object
chunk.55cfcfe8bf5cea239c42.js:12717 card custom:ha-card-weather-conditions SyntaxError: Unexpected token < in JSON at position 0
    at JSON.parse (<anonymous>)
    at HTMLElement.setConfig (ha-card-weather-conditions.js:381)
    at s (chunk.55cfcfe8bf5cea239c42.js:12717)
    at chunk.55cfcfe8bf5cea239c42.js:12717
    at d (chunk.55cfcfe8bf5cea239c42.js:12717)
    at l (chunk.55cfcfe8bf5cea239c42.js:12717)
    at Kt (chunk.55cfcfe8bf5cea239c42.js:11940)
    at HTMLElement.value (chunk.55cfcfe8bf5cea239c42.js:23033)
    at HTMLElement.value (chunk.55cfcfe8bf5cea239c42.js:23460)
    at HTMLElement.<anonymous> (chunk.55cfcfe8bf5cea239c42.js:23033)
l @ chunk.55cfcfe8bf5cea239c42.js:12717
DevTools failed to load SourceMap: Could not load content for https://hassio.kmoneyserver.com/static/69ad52907e8ea109345831d29da5425cb2a55047/dist/serviceWorker.js.map: HTTP error: status code 404, net::ERR_HTTP_RESPONSE_CODE_FAILURE
```

**Comment by zither911** [comment]
I am experiencing the exact same error on a fresh install of home assist with this card

**Comment by zither911** [comment]
chunk.8b29f0c82629b54e915d.js:12852 card custom:ha-card-weather-conditions SyntaxError: Unexpected token < in JSON at position 0
    at JSON.parse (<anonymous>)
    at HTMLElement.setConfig (ha-card-weather-conditions.js:381)
    at s (chunk.8b29f0c82629b54e915d.js:12852)
    at chunk.8b29f0c82629b54e915d.js:12852
    at d (chunk.8b29f0c82629b54e915d.js:12852)
    at l (chunk.8b29f0c82629b54e915d.js:12852)
    at Kt (chunk.8b29f0c82629b54e915d.js:12311)
    at HTMLElement.value (chunk.8b29f0c82629b54e915d.js:23543)
    at HTMLElement.value (chunk.8b29f0c82629b54e915d.js:23978)
    at HTMLElement.t.addEventListener.once (chunk.8b29f0c82629b54e915d.js:23543)

**Comment by r-renato** [comment]
> Hi, the log looks the same except for the one new devtools line at the end. Thanks!
> 
> ```
> ha-card-weather-conditions.js:381  WEATHER-CONDITION-CARD  1.9.5
> chunk.55cfcfe8bf5cea239c42.js:1 Uncaught (in promise) https://hassio.kmoneyserver.com/customcards/github/thomasloven/state-switch.js?track=true
> cloudy.svg:1 Failed to load resource: the server responded with a status of 503 (Service Unavailable)
> cloudy.svg:1 Failed to load resource: the server responded with a status of 404 (Not Found)
> ha-card-weather-conditions.js:381  WEATHER-CONDITION-CARD  1.9.5 error setting right icons path.
> ha-card-weather-conditions.js:381 Object
> chunk.55cfcfe8bf5cea239c42.js:12717 card custom:ha-card-weather-conditions SyntaxError: Unexpected token < in JSON at position 0
>     at JSON.parse (<anonymous>)
>     at HTMLElement.setConfig (ha-card-weather-conditions.js:381)
>     at s (chunk.55cfcfe8bf5cea239c42.js:12717)
>     at chunk.55cfcfe8bf5cea239c42.js:12717
>     at d (chunk.55cfcfe8bf5cea239c42.js:12717)
>     at l (chunk.55cfcfe8bf5cea239c42.js:12717)
>     at Kt (chunk.55cfcfe8bf5cea239c42.js:11940)
>     at HTMLElement.value (chunk.55cfcfe8bf5cea239c42.js:23033)
>     at HTMLElement.value (chunk.55cfcfe8bf5cea239c42.js:23460)
>     at HTMLElement.<anonymous> (chunk.55cfcfe8bf5cea239c42.js:23033)
> l @ chunk.55cfcfe8bf5cea239c42.js:12717
> ha-card-weather-conditions.js:381 Objectcard_config: {alert: {…}, animation: true, language: "en", type: "custom:ha-card-weather-conditions", weather: {…}}__proto__: Object
> chunk.55cfcfe8bf5cea239c42.js:12717 card custom:ha-card-weather-conditions SyntaxError: Unexpected token < in JSON at position 0
>     at JSON.parse (<anonymous>)
>     at HTMLElement.setConfig (ha-card-weather-conditions.js:381)
>     at s (chunk.55cfcfe8bf5cea239c42.js:12717)
>     at chunk.55cfcfe8bf5cea239c42.js:12717
>     at d (chunk.55cfcfe8bf5cea239c42.js:12717)
>     at l (chunk.55cfcfe8bf5cea239c42.js:12717)
>     at Kt (chunk.55cfcfe8bf5cea239c42.js:11940)
>     at HTMLElement.value (chunk.55cfcfe8bf5cea239c42.js:23033)
>     at HTMLElement.value (chunk.55cfcfe8bf5cea239c42.js:23460)
>     at HTMLElement.<anonymous> (chunk.55cfcfe8bf5cea239c42.js:23033)
> l @ chunk.55cfcfe8bf5cea239c42.js:12717
> DevTools failed to load SourceMap: Could not load content for https://hassio.kmoneyserver.com/static/69ad52907e8ea109345831d29da5425cb2a55047/dist/serviceWorker.js.map: HTTP error: status code 404, net::ERR_HTTP_RESPONSE_CODE_FAILURE
> ```

Hi, that sounds like a security/config issue to me.
The card starts and it tries to find the right images path using two paths:
1. <_hostname_>/local/ha-card-weather-conditions/icons/static/cloudy.svg
2. <_hostname_>/local/community/ha-card-weather-conditions//icons/static/cloudy.svg

In your case, it does not find any valid path:
- Failed to load resource: the server responded with a status of 503 (Service Unavailable)
- Failed to load resource: the server responded with a status of 404 (Not Found)

Try to check your system configuration.

Renato

**Comment by zither911** [comment]
Hi,

None of those paths exist.
from the file editor my full path is /config/www/community/ha-card-weather-conditions/icons/static/cloudy.svg

What am i doing wrong?

**Comment by r-renato** [comment]
What is the full address?

**Comment by K-Money** [comment]
> > Hi, the log looks the same except for the one new devtools line at the end. Thanks!
> > ```
> > ha-card-weather-conditions.js:381  WEATHER-CONDITION-CARD  1.9.5
> > chunk.55cfcfe8bf5cea239c42.js:1 Uncaught (in promise) https://hassio.kmoneyserver.com/customcards/github/thomasloven/state-switch.js?track=true
> > cloudy.svg:1 Failed to load resource: the server responded with a status of 503 (Service Unavailable)
> > cloudy.svg:1 Failed to load resource: the server responded with a status of 404 (Not Found)
> > ha-card-weather-conditions.js:381  WEATHER-CONDITION-CARD  1.9.5 error setting right icons path.
> > ha-card-weather-conditions.js:381 Object
> > chunk.55cfcfe8bf5cea239c42.js:12717 card custom:ha-card-weather-conditions SyntaxError: Unexpected token < in JSON at position 0
> >     at JSON.parse (<anonymous>)
> >     at HTMLElement.setConfig (ha-card-weather-conditions.js:381)
> >     at s (chunk.55cfcfe8bf5cea239c42.js:12717)
> >     at chunk.55cfcfe8bf5cea239c42.js:12717
> >     at d (chunk.55cfcfe8bf5cea239c42.js:12717)
> >     at l (chunk.55cfcfe8bf5cea239c42.js:12717)
> >     at Kt (chunk.55cfcfe8bf5cea239c42.js:11940)
> >     at HTMLElement.value (chunk.55cfcfe8bf5cea239c42.js:23033)
> >     at HTMLElement.value (chunk.55cfcfe8bf5cea239c42.js:23460)
> >     at HTMLElement.<anonymous> (chunk.55cfcfe8bf5cea239c42.js:23033)
> > l @ chunk.55cfcfe8bf5cea239c42.js:12717
> > ha-card-weather-conditions.js:381 Objectcard_config: {alert: {…}, animation: true, language: "en", type: "custom:ha-card-weather-conditions", weather: {…}}__proto__: Object
> > chunk.55cfcfe8bf5cea239c42.js:12717 card custom:ha-card-weather-conditions SyntaxError: Unexpected token < in JSON at position 0
> >     at JSON.parse (<anonymous>)
> >     at HTMLElement.setConfig (ha-card-weather-conditions.js:381)
> >     at s (chunk.55cfcfe8bf5cea239c42.js:12717)
> >     at chunk.55cfcfe8bf5cea239c42.js:12717
> >     at d (chunk.55cfcfe8bf5cea239c42.js:12717)
> >     at l (chunk.55cfcfe8bf5cea239c42.js:12717)
> >     at Kt (chunk.55cfcfe8bf5cea239c42.js:11940)
> >     at HTMLElement.value (chunk.55cfcfe8bf5cea239c42.js:23033)
> >     at HTMLElement.value (chunk.55cfcfe8bf5cea239c42.js:23460)
> >     at HTMLElement.<anonymous> (chunk.55cfcfe8bf5cea239c42.js:23033)
> > l @ chunk.55cfcfe8bf5cea239c42.js:12717
> > DevTools failed to load SourceMap: Could not load content for https://hassio.kmoneyserver.com/static/69ad52907e8ea109345831d29da5425cb2a55047/dist/serviceWorker.js.map: HTTP error: status code 404, net::ERR_HTTP_RESPONSE_CODE_FAILURE
> > ```
> 
> Hi, that sounds like a security/config issue to me.
> The card starts and it tries to find the right images path using two paths:
> 
> 1. <_hostname_>/local/ha-card-weather-conditions/icons/static/cloudy.svg
> 2. <_hostname_>/local/community/ha-card-weather-conditions/icons/static/cloudy.svg
> 
> In your case, it does not find any valid path:
> 
> * Failed to load resource: the server responded with a status of 503 (Service Unavailable)
> * Failed to load resource: the server responded with a status of 404 (Not Found)
> 
> Try to check your system configuration.
> 
> Renato

Hi Renato,

I 100% agree that it's a security/config issue from within the work network. However, given that I can't change that and that this is the only card I've encountered that fails in this fashion, that still leaves me with questions:
1) Why does the card itself need to make external calls independent of the component supporting it? The external calls for data are already done by the climacell component and then the card should just be displaying those values, no? I've seen many cards with icons but this is the only one that encounters some kind of security block from my work network. How do other cards with images do it that's different from this method?
2) If there is a valid reason for question 1 and it does need to make independent external calls, why does it fail in such a catastrophic manner when it can't reach them (huuuge red block)? I would expect that if it was looking for an image and it couldn't find it, then it would just default to a blank space, placeholder, or something like that; similar to how a browser will load a webpage and just have placeholders if it can't find the picture. In this way, the card would still be usable since it would be able to display the internal values, it just wouldn't look quite as nice. A far better situation than being completely unable to see the card and having it expand to several times its size, mucking up the layout.

I appreciate your efforts. Thanks!
